src/app/CommandHandler.h

```diff
@@ -190,9 +209,56 @@ class CommandHandler : public Messaging::ExchangeDelegate
     CHIP_ERROR AddClusterSpecificFailure(const ConcreteCommandPath & aCommandPath, ClusterStatus aClusterStatus);
 
     Protocols::InteractionModel::Status ProcessInvokeRequest(System::PacketBufferHandle && payload, bool isTimedInvoke);
-    CHIP_ERROR PrepareCommand(const ConcreteCommandPath & aCommandPath, bool aStartDataStruct = true);
+
+    /**
+     * This will add a new CommandDataIB element into InvokeResponses. It will put the
+     * aResponseCommandPath into CommandPath element within CommandDataIB.
```

Comment:
Is there a way to explain this more behaviorally (e.g. "This method adds the response associated with the `aRequestCommandPath`".)